libmscore/chordline.cpp

```diff
@@ -380,6 +380,8 @@ void ChordLine::updateGrips(EditData& ed) const
 
 void ChordLine::startEdit(EditData& ed)
       {
+      //ChordLine is not editable with drag anchors now, but still initialize ElementEditData for further possible changes
```

Comment:
Same here. Chordlines are editable currently. You can enter one via he Arpeggios and Glissandos palette. The plop, fall, doit, they are Chordlines. Double clicking them will should the anchors.